crates/apub/src/activities/following/accept.rs

```diff
@@ -76,21 +78,47 @@ impl ActivityHandler for AcceptFollowCommunity {
     context: &Data<LemmyContext>,
     request_counter: &mut i32,
   ) -> Result<(), LemmyError> {
-    let person = self
+    let community = self
       .actor
       .dereference(context, local_instance(context), request_counter)
       .await?;
-    let community = self
+    let person = self
       .object
       .actor
       .dereference(context, local_instance(context), request_counter)
       .await?;
     // This will throw an error if no follow was requested
+    let community_id = community.id;
+    let person_id = person.id;
     blocking(context.pool(), move |conn| {
-      CommunityFollower::follow_accepted(conn, person.id, community.id)
+      CommunityFollower::follow_accepted(conn, community_id, person_id)
```

Comment:
Not sure why but these were wrongly named.

Reply:
Maybe bad copy-paste, or just got confused.